site/en/docs/extensions/reference/storage/index.md

```diff
@@ -2,26 +2,66 @@
 api: storage
 ---
 
-## Overview
+## Overview {: #overview }
 
-This API has been optimized to meet the specific storage needs of extensions. It provides the same
-storage capabilities as the [localStorage API][local-storage] with the following key differences:
+The Storage API provides an extension-specific way to persist user data and state. It's similar to the web platform's storage APIs ([IndexedDB][mdn-indexeddb], [localStorage][mdn-localstorage]), but was designed to meet the storage needs of extensions. The following are a few key features: 
 
-- User data can be automatically synced with Chrome sync (using `storage.sync`).
-- Your extension's content scripts can directly access user data without the need for a background
-  page.
-- A user's extension settings can be persisted even when using [split incognito
-  behavior][incognito].
-- It's asynchronous with bulk read and write operations, and therefore faster than the blocking and
-  serial `localStorage API`.
-- User data can be stored as objects (the `localStorage API` stores data in strings).
-- Enterprise policies configured by the administrator for the extension can be read (using
-  `storage.managed` with a [schema][api-storage]).
+- Content scripts can directly access chrome.storage.
+- Stores JSON serializable values as object properties.
+- Storage is asynchronous with bulk read and write operations.
+- Storage data persists when the user clears the cache and browsing history.
+- Available in the service worker, content scripts, and other extension contexts.
+- A user's extension settings persist even when using [split incognito][incognito].
+- Offers an exclusive read-only [managed storage area][manifest-storage] for enterprise policies.
 
-## Manifest
+{% Details %}
+{% DetailsSummary %}
+💡 Can extensions use Window.localStorage?
+{% endDetailsSummary %}
```

Comment:
It's information that some developers may know but beginner developers don't. I like to pack this additional information into expandable sections so that those who want more information can click for it. It also prevents the document from becoming too bulky.

Reply:
Sounds good.